### ISSUE TITLE
Selector syntax errors are confusing

### ISSUE THREAD

**Issue by SidneyNemzer** [tmpvar/jsdom#1940]
### Basic info:

- **Node.js version:** 6.9.4
- **jsdom version:** 11.1.0

### Minimal reproduction case

```js
const { JSDOM } = require("jsdom");

const dom = new JSDOM(`
  <body></body>
`);

dom.window.document.querySelector(':first-child(1)')
```

Error at line 7:
```
[project location]\node_modules\jsdom\lib\jsdom\living\nodes\ParentNode-impl.js:61
    throw new DOMException(DOMException.SYNTAX_ERR, e.message);
    ^
SyntaxError
    at DocumentImpl.<anonymous> ([project location]\node_modules\jsdom\lib\jsdom\living\nodes\ParentNode-impl.js:61:11)
    at DocumentImpl.querySelector ([project location]\node_modules\jsdom\lib\jsdom\utils.js:113:45)
    at Document.querySelector ([project location]\node_modules\jsdom\lib\jsdom\living\generated\ParentNode.js:30:45)
    at Object.<anonymous> ([project location]\[file name].js:[line:column])
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
```

(I've removed some info, replaced with `[some description of removed info]`)

### How does similar code behave in browsers?

The `querySelector` call errors with:

```
Uncaught DOMException: Failed to execute 'querySelector' on 'Document': ':first-child(1)' is not a valid selector.
    at <anonymous>:1:10
```

**Comment by TimothyGu** [comment]
This should be fixed also when #1939 is fixed.

**Comment by SidneyNemzer** [comment]
Closing this since #1939 has been closed
